server/push/test/test_push.js

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2017 The Absolute Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the 'License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import assert from 'assert';
+import {describe, it} from 'mocha';
+import * as pushSender from '../controller/send_push';
+
+const TEST_PUSH_TOKENS_PC_CHROME = {
+  endpoint: 'https://fcm.googleapis.com/fcm/send/fvkYiDvVn_k:APA91bH8h4uZK4F' +
```

Comment:
IMHO, we should implement mock-push service(?) if possible instead of actual service.
(Because the system is down or changed or something happens I think)
But I think this test is good for now. Let's improve it incrementally.

Reply:
Yes right, but we need to check positive case.
So my plan is register push token as valid (to get 201 from push service) one. but not actual used for developing this project and not used for real user 
(for PC browser, push is not recieved when browser is not launched or connected to server.
for mobile browser, device will be old one that is not used anymore)